### PR TITLE
Better chapter anchors

### PR DESCRIPTION
**Pull request by mbrandonw**
Improves the UX of the chapter anchors by _not_ jumping to the text content when on mobile, only for desktop.

### CODE REVIEW

File: Sources/PointFree/Episode/Show.swift [hunk 1]
```diff
@@ -108,15 +108,23 @@ private let episodeTocView = View<(blocks: [Episode.TranscriptBlock], isEpisodeV
   )
 }
 
-private func timestampLinkAttributes(_ timestamp: Int) -> [Attribute<Element.A>] {
-  return [
-    href("#t\(timestamp)"),
+private func timestampLinkAttributes(timestamp: Int, useAnchors: Bool) -> [Attribute<Element.A>] {
```
Comment (mbrandonw) [inline Sources/PointFree/Episode/Show.swift]: now we can customize timestamp links based on whether or not we also want to anchor them.

File: Sources/PointFree/Episode/Show.swift [hunk 2]
```diff
@@ -137,19 +145,34 @@ private let tocChapterView = View<(title: String, timestamp: Int, isEpisodeViewa
...
+private let tocChapterLinkView = View<(title: String, timestamp: Int, active: Bool)> { title, timestamp, active -> [Node] in
   if active {
-    return a(
-      timestampLinkAttributes(timestamp) +
-        [`class`([Class.pf.colors.link.green, Class.type.textDecorationNone, Class.pf.type.body.regular])],
-      [text(title)]
-    )
+    return
+      [
+        div([`class`([Class.hide(.mobile)])], [
```
Comment (mbrandonw) [inline Sources/PointFree/Episode/Show.swift]: we render two chapter links: one for mobile without anchors, one for desktop with anchors.

File: Sources/PointFree/TODO.swift [hunk 3]
```diff
@@ -309,6 +309,10 @@ public func onclick<T>(javascript: StaticString) -> Attribute<T> {
   return .init("onclick", "javascript:\(javascript)")
 }
 
+public func onclick<T>(unsafeJavascript: String) -> Attribute<T> {
+  return .init("onclick", "javascript:\(unsafeJavascript)")
```
Comment (mbrandonw) [inline Sources/PointFree/TODO.swift]: new helper for being explicit when we are doing potentially unsafe things. without this I needed to create two separate scripts for the timestamp links, one that prevents default and one that doesnt.

**Comment by mbrandonw** [comment]
I also got weird small changes in the numbers on the privacy page :/

File: Sources/PointFree/Episode/Show.swift (hunk 2 above)
Comment (stephencelis) [reply Sources/PointFree/Episode/Show.swift]: Easy peasy.

**Comment by stephencelis** [comment]
I got the same changes :/ I think it's when my OS got upgraded.

**Pull request merged**